roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/entity/StreetSectionTest.java

```diff
@@ -15,18 +15,31 @@
 
 public class StreetSectionTest {
     @Test
-    public void firstCarCouldEnterNextSection() throws Exception {
+    public void firstCarCouldEnterNextSection_firstCarNotOnExitPoint() throws Exception {
         IStreetSection streetSectionMock = mock(StreetSection.class);
         when(streetSectionMock.firstCarCouldEnterNextSection()).thenCallRealMethod();
 
         // first car not on exit point
         when(streetSectionMock.isFirstCarOnExitPoint()).thenReturn(false);
```

Comment:
this line belongs to the arrange part, make sure you visually split all three parts: Arrange Act Arrange

Reply:
you can extract the first two lines into one method, which can be used in multiple tests: reduce point of changes!

`public IStreetSection createStreetSection(){
 IStreetSection streetSectionMock = mock(StreetSection.class);
when(streetSectionMock.firstCarCouldEnterNextSection()).thenCallRealMethod();
return streetSectionMock;
}
`
and use it in the test:
streetSectionMock = createStreetSection();